### PR TITLE
Add Support For Close Clients and Server

### PR DESCRIPTION
**Pull request by samratashok87**
Implemented keywords for closing server and client.
Also keyword is added to manually switch client and server. 
Have also included the acceptance test and updated the keyword documentation.

### CODE REVIEW

File: src/Rammbock/core.py [hunk 1]
```diff
@@ -1028,3 +1029,44 @@ def get_server_unread_messages_count(self, server_name=None):
         """
         server = self._servers.get(server_name)
         return server.get_messages_count_in_buffer()
+
+    def close_client(self, name=None):
+        """Closes the client connection based on the `client_name`.
+
+        If no name provided it will close the current active connection.
+        You have to explicitly Switch Client after close when sending or
```
Comment (jussimalinen) [inline src/Rammbock/core.py]: Other keywords in the docs should be in backticks: `Switch Client`. The generated api docs will then contain working links to that kw,

**Comment by jussimalinen** [comment]
Nice! If you could update the docs to surround the keyword names in backticks, we would get links to them in the docs. Other than that, this looks ready.

**Comment by samratashok87** [comment]
Have updated the documentation with the backtick for the referred keywords.

**Comment by jussimalinen** [comment]
Brilliant! Thanks!

**Pull request merged**